osgp/protocol-adapter-dlms/osgp-dlms/src/test/java/org/opensmartgridplatform/dlms/services/Protocol.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * SPDX-FileCopyrightText: Copyright Contributors to the GXF project
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensmartgridplatform.dlms.services;
+
+import org.opensmartgridplatform.dlms.objectconfig.PowerQualityRequest;
+
+public enum Protocol {
+  DSMR_2_2("DSMR", "2.2", 20, 5, 14, 0, 0),
+  DSMR_4_2_2("DSMR", "4.2.2", 43, 27, 14, 16, 10),
+  SMR_4_3("SMR", "4.3", 44, 27, 14, 16, 11),
+  SMR_5_0_0("SMR", "5.0.0", 49, 27, 14, 27, 18),
+  SMR_5_1("SMR", "5.1", 49, 27, 14, 27, 18),
+  SMR_5_2("SMR", "5.2", 50, 27, 14, 27, 19),
+  SMR_5_5("SMR", "5.5", 51, 27, 14, 27, 19);
+
+  private final String name;
+  private final String version;
+  private final int nrOfCosemObjects;
```

Comment:
I find this name confusing, it seems to be the total number of objects for this version of (D)SMR. But it's meant to be the expected number of power quality related objects.
Maybe this class is not the right place for this information? Or is this class intended to become the place for all protocol related specifics?